composer.json

```diff
@@ -25,12 +25,14 @@
         "symfony/yaml": "^5.4 || ^6.3 || ^7.0"
     },
     "require-dev": {
-        "doctrine/annotations": "^1.8.0 || ^2.0",
+        "doctrine/annotations": "^1.13.1 || ^2.0",
         "doctrine/data-fixtures": "^1.7",
+        "doctrine/dbal": "^3.7",
```

Comment:
Added as an explicit dev dependency to call it out versus relying on transient dependencies.

DBAL 3.7 is set as the minimum due to the DoctrineBundle minimum being set so high in #248, it does not support older DBAL versions anymore so this range would need to be opened back up to a DBAL 2 friendly version to be able to test it if so desired.

Reply:
> DBAL 3.7 is set as the minimum due to the DoctrineBundle minimum being set so high in https://github.com/liip/LiipTestFixturesBundle/pull/248

My bad, I didn't checked that this version was only 4 months-old: https://packagist.org/packages/doctrine/doctrine-bundle#2.11.0

Can we lower it to increase the supported versions for users?

Reply:
With a couple of tweaks I was able to get back to allowing it to install DBAL 2.13.1.  So that should keep DBAL 2 covered for the 2.x branch here.  Now to fix the bugs that exposed 😆